scripts/build_renewable_profiles.py

```diff
@@ -60,7 +60,6 @@
     **Source:** `GEBCO <https://www.gebco.net/data_and_products/images/gebco_2019_grid_image.jpg>`_
 
 - ``resources/natura.tiff``: confer :ref:`natura`
-- ``resources/country_shapes.geojson``: confer :ref:`shapes`
```

Comment:
This is still present in `Snakefile` , right?

Reply:
Yes, and also used